pywwt/core.py

```diff
@@ -435,6 +437,28 @@ def add_line(self, points=None, **kwargs):
             line.add_point(points)
         return line
 
+    def add_fov(self, telescope, center=None, rotate=0 * u.rad, **kwargs):
+        """
+        Add a telescope's field of view (FOV) to the current view.
```

Comment:
It would be good to mention how to find out the list of available FOVs.

Reply:
Done.